src/transformers/models/convbert/configuration_convbert.py

```diff
@@ -85,11 +85,11 @@ class ConvBertConfig(PretrainedConfig):
     Example:
 
     ```python
-    >>> from transformers import ConvBertModel, ConvBertConfig
+    >>> from transformers import ConvBertConfig, ConvBertModel
 
     >>> # Initializing a ConvBERT convbert-base-uncased style configuration
     >>> configuration = ConvBertConfig()
-    >>> # Initializing a model from the convbert-base-uncased style configuration
+    >>> # Initializing a model (with random weights) from the convbert-base-uncased style configuration
```

Comment:
Could we add a few new empty lines in this doc example. See for example the BertConfig file 🙏

Reply:
@ydshieh Added the empty lines. Should be good to go now